src/eHealth.couchQuery/services/lucene-query-factory.js

```diff
@@ -104,9 +105,12 @@ angular
                   .then(function(response) {
                     return response.data;
                   });
-              }, initialParams);
+              }, initialParams, initialOptions);
             } else {
               initialParams.q = q;
+              angular.extend({
```

Comment:
I dont know what you mean by the destination variable is anonymous.

I set unique further down, because i thought `unique` should only be set for the lucene search and not the couchdb view. I understand its a little confusing, but its the same with params atm.